### PR TITLE
ISPN-13884 RemoteStore should not allow segmentation if groups are co…

### PR DESCRIPTION
**Pull request by wburns**
…nfigured

https://issues.redhat.com/browse/ISPN-13884

### CODE REVIEW

File: persistence/remote/src/main/java/org/infinispan/persistence/remote/logging/Log.java [hunk 1]
```diff
@@ -57,4 +57,7 @@ public interface Log extends BasicLogger {
 
    @Message(value = "Segmentation is not supported for a RemoteStore when the configured key media type %s does not match the remote servers key media type %s", id = 10011)
    CacheConfigurationException segmentationRequiresEqualMediaTypes(MediaType cacheMediaType, MediaType serverMediaType);
+
+   @Message(value = "The RemoteCacheStore cannot be segmented when grouping is enabled")
```
Comment (jabolina) [inline persistence/remote/src/main/java/org/infinispan/persistence/remote/logging/Log.java:61]: `id` is not needed?

Comment (wburns) [reply persistence/remote/src/main/java/org/infinispan/persistence/remote/logging/Log.java:61]: Which `id`? Are you referring to a cache name for example? Unfortunately, our other messages don't have that either.

Comment (jabolina) [reply persistence/remote/src/main/java/org/infinispan/persistence/remote/logging/Log.java:61]: The annotation property, for example in line 58 with `id = 10011`

https://github.com/infinispan/infinispan/blob/db6de87be1c9edbcfb6e354027e39482da6b298d/persistence/remote/src/main/java/org/infinispan/persistence/remote/logging/Log.java#L58

Comment (wburns) [reply persistence/remote/src/main/java/org/infinispan/persistence/remote/logging/Log.java:61]: Ah, yes. I need the id number 👍

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Merged, thanks @wburns

**Pull request merged**